packages/integration-boilerplate-node/tsconfig.json

```diff
@@ -23,7 +23,7 @@
     "experimentalDecorators": true,
     "resolveJsonModule": true
   },
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "./node_modules/@saasquatch/schema/index.d.ts"],
```

Comment:
Surely there is a better way than this, but I couldn't figure out how to get TypeScript to recognize the SaaSquatch namespace from the npm package.